vendor/wp-now/src/wp-now.ts

```diff
@@ -633,6 +633,24 @@ async function mountInternalMuPlugins( php: PHP, options: WPNowOptions ) {
 		}, 1, 3);
 		`
 	);
+
+	php.writeFile(
+		path.posix.join( PLAYGROUND_INTERNAL_MU_PLUGINS_FOLDER, '0-tmp-fix-hide-plugins-spinner.php' ),
+		`<?php
+			// This is a temporary fix for a page-optimize bug that causes spinner icons to show all the time in the plugins list auto-update column
+			// @see https://github.com/Automattic/wpcomsh/pull/975
+			add_action( 'admin_enqueue_scripts', 'wpcomsh_patch_auto_update_spinner_style', 999 );
+			function wpcomsh_patch_auto_update_spinner_style() {
```

Comment:
Would it make sense to rename this function to avoid potential conflicts for someone working on wpcomsh development in Studio?

Also, link points to private repository, so let's remove it.

Reply:
>Would it make sense to rename this function to avoid potential conflicts for someone working on wpcomsh development in Studio?

I intentionally decided to keep the sema name, to simplify for other developers to find this function in Studio, since it's the same fix.

>Also, link points to private repository, so let's remove it.

Ok

Reply:
It might be safer to use our own prefix, e.g., `studio_patch_auto_update_spinner_style`.

Reply:
I see, updated